public/app/features/alerting/state/alertDef.ts

```diff
@@ -207,6 +207,25 @@ function getAlertAnnotationInfo(ah: any) {
   return '';
 }
 
+// Copy of getAlertAnnotationInfo, used in annotation tooltip
+function getAlertAnnotationText(annotationData: any) {
+  // backward compatibility, can be removed in grafana 5.x
```

Comment:
Seems like this shouldn't be here?

Reply:
I mean, it's true... 😂

Reply:
probably the entire copy function shouldn't be here 😅 but unsure how to deal with its usage in graph rn..

Reply:
LOL! I think a little copypasta is great for the first move!